feedback-module/src/components/common/Module.js

```diff
@@ -40,7 +92,50 @@ function Module() {
       className={MODULE_CONTAINER_STYLE}
     >
       <Header />
-      {screen}
+      {screen.titleInverse && (
+        <Grid className={`bg-primary ${SCREEN_CONTAINER_STYLE}`}>
+          <p
+            className={`${H1_WHITE_STYLE}`}
+            dangerouslySetInnerHTML={{ __html: screen.titleInverse }}
```

Comment:
Why are you using dangerouslySetInnerHTML and not just putting the title inside the children of the paragraph component?